File: restaurantReviews.client/src/models/Restaurant.js

export class Restaurant{
  constructor(data = {}){
    this.id = data.id
    this.creatorId = data.creatorId
    this.createdAt = data.createdAt
    this.updatedAt = data.updatedAt
    this.name = data.name
    this.location = data.location
    this.creator = data.creator || {}
    this.picture = 'https://images.unsplash.com/photo-1552566626-52f8b828add9?ixid=MnwxMjA3fDB8MHxzZWFyY2h8Nnx8cmVzdGF1cmFudHxlbnwwfHwwfHw%3D&ixlib=rb-1.2.1&w=1000&q=80'
  }
}

File: restaurantReviews/Models/Restaurant.cs
using System;
using restaurantReviews.Models;

namespace week11practice.Models
{
  public class Restaurant
  {
    public int Id { get; set; }
    public string CreatorId { get; set; }
    public DateTime CreatedAt { get; set; }
    public DateTime UpdatedAt { get; set; }
    public string Name { get; set; }
    public string Location { get; set; }
    public Profile Creator { get; set; }

  }
}